docs/pages/reference/networking.mdx

```diff
@@ -223,7 +227,14 @@ agents to the public internet.
 Some Teleport services listen for traffic to one of their proxied resources,
 meaning that you can expose ports on that service's host directly to clients.
 This is useful when you need to connect to resources directly if the Proxy
-Service becomes unavailable.
+Service becomes unavailable. 
+
+<Admonition
+  type="tip"
+  title="Note"
+>
+  Only self-hosted Teleport clusters support Teleport services exposing ports.
```

Comment:
What does this mean?

Reply:
Since you can't dial to the auth server you can't have the SSH service or other services exposing ports. As an example you can't expose the SSH port 3022 as the means of connecting.

Reply:
Gotcha. How about something like this?

In Teleport Cloud, the Auth and Proxy Services run in Teleport-owned infrastructure.
For this reason, Teleport Cloud customers must connect their resources via reverse tunnels.
Exposing ports for direct dial is only supported in self-hosted deployments.

Reply:
updated!